src/components/ContactList/ContactList.tsx

```diff
@@ -1,16 +1,17 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import styles from './ContactList.module.css';
-import { useDispatch, useSelector } from 'react-redux';
+import { useSelector } from 'react-redux';
 import { deleteContact, getContacts } from '../../redux/actions';
 import { deleteContactService } from '../../serviсes/deleteContactService';
 import * as phonebookSelectors from '../../redux/selectors';
+import { useAppDispatch } from '../../redux/store'
+import ContactListProps from '../../interfaces/ContactList.interface';
```

Comment:
move `ContactListProps ` into this file

Reply:
All interfaces including this one have been moved to their own files.